documents/adr/0011-secrets-management.md

```diff
@@ -0,0 +1,29 @@
+# 11. Secrets management
```

Comment:
Good synopsis in the ADR. Overall, the idea of doing this vs. manually creating secrets seems like a big improvement.